Automattic/jetpack
# Fix/creator goes to blank page after jetpack install

## Proposed changes:

* This diff fixes an issue where Jetpack Creator redirects to the wrong page when selecting "start for free" from My Jetpack AND the Jetpack plugin is not activated already.
* 
## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






* It is simplest to test this issue on your local Jetpack install instead of with Jetpack beta, because you must deactivate the Jetpack plugin.
* To reproduce this issue without the patch, make sure the main Jetpack plugin is not active
* From My Jetpack, click "Learn More" on the Creator card
* Select "start for free"
* After a while, you will be redirected to ?page=my-jetpack#/term=creator - this is incorrect and the screen will be blank
* Now, apply this PR  and deactivate the Jetpack plugin again
* From My Jetpack, test "start for free" from the creator card
* Now, you should redirect to ?page=jetpack#/term=creator after the Jetpack plugin is activated - this is correct.
